lib/ecto/adapters/postgres/connection.ex

```diff
@@ -1619,26 +1619,24 @@ if Code.ensure_loaded?(Postgrex) do
     end
 
     defp column_type(:identity, opts) do
-      start_value = [Keyword.get(opts, :start_value)]
-      increment = [Keyword.get(opts, :increment)]
+      start_value = Keyword.get(opts, :start_value)
+      increment = Keyword.get(opts, :increment)
+      cache = Keyword.get(opts, :cache)
+      clause = Keyword.get(opts, :clause, :by_default) |> to_string |> String.upcase |> String.split("_") |> Enum.join(" ")
```

Comment:
I would prefer to make this as a string: `clause: "BY DEFAULT"`, so there is less magic around it. Perhaps we should call the option `:generated` too.

Reply:
Yeah helps to clear some crud.